packages/core/package.json

```diff
@@ -107,7 +107,11 @@
     "node": "^16.13.0 || ^18.12.0"
   },
   "eslintConfig": {
-    "extends": "@silverhand"
+    "extends": "@silverhand",
+    "rules": {
+      "complexity": ["error", 11],
+      "default-case": "off"
+    }
```

Comment:
will add to the common config:
- complexity 7 is too small for complex biz logic since even `??` will add one. default is 20 so we are good i think.
- `default-case` is useless when we enforce exhaustive switch cases